core/src/main/java/ai/timefold/solver/core/impl/solver/ConsumerSupport.java

```diff
@@ -42,6 +44,27 @@ void consumeIntermediateBestSolution(Solution_ bestSolution, BooleanSupplier isE
         }
     }
 
+    // Called on the Solver thread.
+    void consumeInitializedSolution(Solution_ initializedSolution) {
+        if (initializedSolutionConsumer != null) {
+            // TODO - Do we need to sync it?
```

Comment:
Why would we need to?

Reply:
I don't think we need to sync it, unless I'm missing something.